ansible_base/authentication/authenticator_plugins/local.py

```diff
@@ -69,5 +78,8 @@ def authenticate(self, request, username=None, password=None, **kwargs):
                     "is_superuser": user.is_superuser,
                 },
             )
+            logger.info(f"Successful login for user: {username} {auth_log_headers}")
+        else:
+            logger.info(f"Failed login for user: {username} {auth_log_headers}")
```

Comment:
We are logging the auth headers here again, I think we can drop it out of these messages.

Reply:
I think its valuable to have the status and the headers in a single log entry. It lets you easily determine where the successes/failures are coming from. I think we can probably drop the `attempt` log to debug